### PR TITLE
Enable parsing of Types

### PR DESCRIPTION
**Pull request by SGamoff**
When I tried to process the file with the types block I got an error:
`if isinstance(lopen[0], Container):`
`IndexError: list index out of range`
I decided to fix the code as in the fix [#29](https://github.com/peakwinter/python-nginx/pull/29) If you think that this is necessary, then I would be happy to help.

#33

### CODE REVIEW

File: nginx.py [hunk 1]
```diff
@@ -436,49 +436,56 @@ def loads(data, conf=True):
...
-        m = re.compile(r'^\s*location\s*(.*?)\s*{', re.S).search(data[index:])
+        m = re.compile(r'^\s*location\s*([^;]*?)\s*{', re.S).search(data[index:])
         if m:
             l = Location(m.group(1))
             lopen.insert(0, l)
             index += m.end()
             continue
 
-        m = re.compile(r'^\s*if\s*(.*?)\s*{', re.S).search(data[index:])
+        m = re.compile(r'^\s*if\s*([^;].*?)\s*{', re.S).search(data[index:])
```
Comment (fulder) [inline nginx.py]: The `.` is still in this `if` block and the following `upstream`, `geo`, `map`, `limit_exception` btw. Forgotten to write it in the previous review.

Comment (fulder) [reply nginx.py]: Other than that this looks good to me. Good job! :smile:

Comment (SGamoff) [reply nginx.py]: And for me too @fulder Thanks again for you review.

Comment (fulder) [reply nginx.py]: You should probably remove the `.` from the other places as well.

Comment (SGamoff) [reply nginx.py]: I did it hard but I think I did it. ))

Comment (fulder) [reply nginx.py]: looks good now +1 from me to merge

**Pull request merged**